### ISSUE TITLE
[a11y] extended permissions checkbox is missing proper label

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4469]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Should have a proper label.

### Actual behaviour
While there is a label element, it does not have a real content but only a `title` attribute.
Should be normal `<label>content</label>` or `aria-label` attribute.

<img width="876" height="586" alt="Image" src="https://github.com/user-attachments/assets/f1fdbf7e-8fe3-4955-9c6d-2781f45545ef" />
